src/phoebus_driver.cpp

```diff
@@ -159,9 +139,7 @@ void PhoebusDriver::PostInitializationCommunication() {
     auto &tl = async_region_2[i];
     auto &sc = pmb->meshblock_data.Get();
 
-    auto prolongBound = tl.AddTask(none, parthenon::ProlongateBoundaries, sc);
-
-    auto set_bc = tl.AddTask(prolongBound, parthenon::ApplyBoundaryConditions, sc);
+    auto set_bc = tl.AddTask(none, parthenon::ApplyBoundaryConditions, sc);
```

Comment:
`parthenon::ApplyBoundaryConditions` (which I believe is filling boundaries on the fine level) is already taken care of by the tasks added by `AddBoundaryExchangeTasks` (although applying the boundary conditions twice shouldn't hurt anything I guess).